requirements.txt

```diff
@@ -0,0 +1,6 @@
+scylla-driver==3.25.7
+six==1.16.0
+geomet==0.2.1.post1
+PyYAML==6.0
```

Comment:
Do we need PyYAML for cqlsh? geomet?

Reply:
yes, for the python-driver
and this list is the exact versions that gonna be bundled